lib/model/txproposal.js

```diff
@@ -171,8 +182,7 @@ TxProposal.prototype.getBitcoreTx = function() {
 };
 
 TxProposal.prototype.getNetworkName = function() {
-  var someAddress = this.toAddress || this.outputs[0].toAddress;
-  return Bitcore.Address(someAddress).toObject().network;
+  return Bitcore.Address(this.changeAddress.address).toObject().network;
```

Comment:
no sure we should relay on change Address to be present.  maybe we should add change address to the  `||` on someAddress.